docker-compose.yml

```
@@ -0,0 +1,50 @@
+      - "6379"
+    ports:
+      - "6379:6379"
+    volumes:
+      - ./data:/data
+    deploy:
+      resources:
+        limits:
+          memory: 80M
+
+volumes:
+  grafana-storage:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)